ci-scripts/ci-runner.bash

```diff
@@ -5,6 +5,7 @@ CI_FOLDER=""
 CI_GENERIC=0
 CI_TUTORIAL=0
 CI_EXITCODE=0
+CI_LOCAL=0
```

Comment:
This is equivalent to `CI_GENERIC`, isn't it?

Reply:
Not exactly because in the `CI_GENERIC` case no added regression tests are actually executed. Only the public unit tests are run.

Reply:
So you run the unit tests with the generic config and the regression tests with the CSCS config?

Reply:
Yes, since regression tests including `fulen` should be tested.

Reply:
That looks like a workaround just for Fulen. I don't agree. Either we enable it fully as for all other systems or run with only the public settings overall. We might need to increase the priority of the Jenkins jobs? A special account? @victorusu What do you think?

Reply:
Guys, @victorusu @teojgo, what do we do with this?